RoadMap_Movie/Tests/UITests/RoadMapMovieUITests.swift

```diff
@@ -0,0 +1,48 @@
+// RoadMapMovieUITests.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+import XCTest
+
+final class RoadMapMovieUITests: XCTestCase {
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests
+        // before they run. The setUp method is a good place to do this.
+    }
+
+    override func tearDownWithError() throws {}
+
+    func testExample() throws {
+        let app = XCUIApplication()
+        app.launch()
+
+        app.alerts.buttons["Ok"].tap()
+
+        let tablesQuery = app.tables
+        tablesQuery.staticTexts["Побег из Шоушенка"].tap()
+
+        let movieButton = app.navigationBars["Movie"].buttons["Movie"]
+        movieButton.tap()
+
+        let staticText = tablesQuery.staticTexts["Крёстный отец 2"]
+        staticText.swipeUp()
+        movieButton.tap()
+        app.staticTexts["Популярное"].tap()
+        tablesQuery.staticTexts["Аватар: Путь воды"].tap()
+        movieButton.tap()
```

Comment:
```suggestion
        tablesQuery.staticTexts["Побег из Шоушенка"].tap()

        let movieButton = app.navigationBars["Movie"].buttons["Movie"]
        movieButton.tap()

        let staticText = tablesQuery.staticTexts["Крёстный отец 2"]
        staticText.swipeUp()
        movieButton.tap()
        app.staticTexts["Популярное"].tap()
        tablesQuery.staticTexts["Аватар: Путь воды"].tap()
        movieButton.tap()
```
все стринги в константы